README.md

```diff
@@ -44,9 +44,9 @@ from frequenz.client.common.metric import Metric
 from frequenz.client.reporting import ReportingApiClient
 
 # Change server address if needed
-SERVICE_ADDRESS = "reporting.api.frequenz.com:443"
+SERVICE_URL = "grpc://reporting.api.frequenz.com:443?ssl=true"
 API_KEY = open('api_key.txt').read().strip()
-client = ReportingApiClient(service_address=SERVICE_ADDRESS, key=API_KEY)
+client = ReportingApiClient(service_url=SERVICE_URL, key=API_KEY)
 ```
 
 Besides the microgrid_id, component_ids, and metrics, start, and end time,
```

Comment:
We should also (have already) update the doc part on the CLI in L112.

Reply:
> doc part on the CLI in L112.

Reply:
This line https://github.com/frequenz-floss/frequenz-client-reporting-python/pull/95/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R112

Reply:
I will update this in a separate PR, as I also want to update `main` to the "new" default.